tomcat/project.clj

```diff
@@ -17,9 +17,8 @@
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
   :dependencies [[org.clojure/clojure "1.9.0"]
-                 [org.apache.tomcat.embed/tomcat-embed-logging-juli "8.0.52"]
```

Comment:
Did `org.apache.tomcat.embed/tomcat-embed-logging-juli` get subsumed or is it unnecessary now?

Reply:
According to https://stackoverflow.com/a/45428044/15861555 for tomcat newer than  `8.5.2` it's not required anymore.